From cb83c14628bea42d99236c3c9584abf43196b901 Mon Sep 17 00:00:00 2001
From: Peter Korsgaard <peter@korsgaard.com>
Date: Fri, 11 Jan 2019 09:17:42 +0100
Subject: [PATCH 40/55] Cryptlib: fix build on 32bit ARM

Pass MDE_CPU_ARM, similar to how it is done for the other supported
architectures, otherwise the build fails in:

Cryptlib/Include/OpenSslSupport.h:55:2: error:
 #error Unknown target architecture

Signed-off-by: Peter Korsgaard <peter@korsgaard.com>
---
 Cryptlib/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Cryptlib/Makefile b/Cryptlib/Makefile
index 2aa5695..5c098f3 100644
--- a/Cryptlib/Makefile
+++ b/Cryptlib/Makefile
@@ -19,6 +19,9 @@ endif
 ifeq ($(ARCH),aarch64)
 	CFLAGS	+= -DMDE_CPU_AARCH64
 endif
+ifeq ($(ARCH),arm)
+	CFLAGS	+= -DMDE_CPU_ARM
+endif
 LDFLAGS		= -nostdlib -znocombreloc
 
 TARGET		= libcryptlib.a
-- 
2.17.1

